android/app/build.gradle

```diff
@@ -101,11 +101,14 @@ android {
     }
     defaultConfig {
         applicationId "com.reactnativeapp"
-        minSdkVersion 16
+        minSdkVersion 18
         targetSdkVersion rootProject.ext.targetSdkVersion
         versionCode 1
         versionName "1.0"
         multiDexEnabled true
+        testBuildType System.getProperty('testBuildType', 'debug')  //this will later be used to control the test apk build type
```

Comment:
Was this in consultation with the Android devs?

Reply:
Sorry, this is copy pasted from `detox` docs, and it is confusing. I will remove the comment.